testing/trino-testing/src/main/java/io/trino/testing/MultisetAssertions.java

```diff
@@ -22,19 +23,25 @@
 import static com.google.common.collect.ImmutableList.toImmutableList;
 import static java.lang.String.format;
 import static java.lang.String.join;
-import static org.assertj.core.api.Fail.fail;
+import static org.assertj.core.api.Assertions.assertThat;
 
 public final class MultisetAssertions
 {
     private MultisetAssertions() {}
 
     public static void assertMultisetsEqual(Multiset<?> actual, Multiset<?> expected)
+    {
+        List<String> mismatchReport = mismatchMultisets(actual, expected);
+        assertThat(mismatchReport).as("Expected: \n\t\t" + join(",\n\t\t", mismatchReport)).isEmpty();
+    }
+
+    public static List<String> mismatchMultisets(Multiset<?> actual, Multiset<?> expected)
```

Comment:
Can we move this method to `TestHiveGlueMetastoreAccessOperations`, or mark as `@Deprecated`? I don't think we want to encourage this method.